cocotb/utils.py

```diff
@@ -196,7 +202,16 @@ def unpack(ctypes_obj, string, bytes=None):
         :exc:`ValueError`: If length of *string* and size of *ctypes_obj*
             are not equal.
         :exc:`MemoryError`: If *bytes* is longer than size of *ctypes_obj*.
+
+    .. deprecated:: 1.5
+        Its functionality is now built into Python 3 as ``memoryview(ctypes_obj)[:bytes] = string``.
+        You may also be looking for :meth:`from_buffer_copy <python:ctypes._CData.from_buffer_copy>`
+        if you are trying to convert :class:`bytes` to a :class:`python:ctypes.Structure` instance;
```

Comment:
Does this really need the `python:` bit?

Reply:
Nope.